1.8/usage/service-discovery/haproxy-adminrouter.md

```diff
@@ -0,0 +1,107 @@
+---
+post_title: Configuring HAProxy in Front of Admin Router
+nav_title: HAProxy in Front of Admin Router
+menu_order: 400
+---
+
+You can use HAProxy to set up an HTTP proxy in front of the DC/OS [Admin Router](/docs/1.8/overview/concepts/#adminrouter).
```

Comment:
What is the reason for doing this? Might be helpful to discuss that. I am not sure, but I think this is to prevent warnings from self-signed certificates? But perhaps there are other reasons to do this. It might be good to discuss the reasons for doing this. If I am correct and it is for the self-signed certificate issue, then I can add a link to this page from my other section. I am guessing that the discussion of this issue could be somewhat complex because we want to single-source this content from open to enterprise. I am not super familiar with the open source offering, but I believe that the use case is somewhat similar there, where they have a "snake-oil certificate", but I am not up on the details at this time.